jest.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
 
   collectCoverage: true,
   coverageDirectory: `./test/unit/coverage`,
-  coverageReporters: [`lcov`],
+  coverageReporters: [`lcov`, `text`], //change 'text' to 'text-summary' for summarized output
```

Comment:
can you share what `text` and `text-summary` look like so we can make a call on what we want the default to be?

Reply:
With just "text" (describes every file):
![image](https://user-images.githubusercontent.com/11528441/59205761-862a5f80-8b71-11e9-9417-e86aa485ced3.png)


With "text-summary":
![image](https://user-images.githubusercontent.com/11528441/59205857-bffb6600-8b71-11e9-97dc-b13077b10d85.png)

Reply:
What would you prefer?

Reply:
cooooooooooool. maybe both?

Reply:
How Long does a full 'text' report get?

Reply:
It gets pretty long, 135 lines